lib/my_obfuscate.rb

```diff
@@ -6,11 +6,13 @@
 # Class for obfuscating MySQL dumps. This can parse mysqldump outputs when using the -c option, which includes
 # column names in the insert statements.
 class MyObfuscate
-  attr_accessor :config, :globally_kept_columns, :fail_on_unspecified_columns, :database_type, :scaffolded_tables
```

Comment:
With this change, I get `undefined method `fail_on_unspecified_columns=' for #<MyObfuscate:0x000000009cce58> (NoMethodError)`